### PR TITLE
Font, Tab Color Changes to Theme

### PR DESCRIPTION
**Pull request by leeberg**
* Allow Specification of Font in Themes
* Change Tab Color and Font Slighty to be more uniform

### CODE REVIEW

File: src/UniversalDashboard/Themes/Default.psd1 [hunk 1]
```diff
@@ -108,7 +111,7 @@ $AlternativeBackgroundColor3 = $BackgroundColorBright
         }
 
         UDTabIndicator = @{
-            IndicatorColor = $SecondaryColor
+            IndicatorColor = $PrimaryColorDark
```
Comment (leeberg) [inline src/UniversalDashboard/Themes/Default.psd1]: This was the red under the tab

File: src/UniversalDashboard/Themes/Default.psd1 [hunk 2]
```diff
@@ -83,12 +85,13 @@ $AlternativeBackgroundColor3 = $BackgroundColorBright
             BackgroundColor = $AlternativeBackgroundColor3
             Height = 'auto'
             BoxShadow = 'unset'
+            FontFamily = $FontFamily
 
         }
```
Comment (leeberg) [inline src/UniversalDashboard/Themes/Default.psd1]: Had to Set FontFamily in two places to make this stick - I thought doing it on UDDashboard would be enough but nope.

**Comment by adamdriscoll** [comment]
@leeberg  Can you update the changelog.md?

**Comment by leeberg** [comment]
Added Change Log - annnnnnnd merging!

**Pull request merged**